### ISSUE TITLE
[Server bug] Error logs in console on player join

### ISSUE THREAD

**Issue by itsdinkd** [MoonflowerTeam/pollen#38]
### What version are you seeing the problem on?

1.18.2

### Describe the issue

```markdown
1. Start Server
2. Player Joins
3. See console
```

### latest.log

```shell
https://gist.github.com/itsdinkd/b99260c1747505f32460b03f36ce6744

log was too big
```

**Comment by jacksonhardaway** [comment]
Make sure the player trying to join has Pollen, it seems like they don't have it installed.

**Comment by itsdinkd** [comment]
Everyone joining is making this error occur. They all have pollen as it's
in the client modpack that was just released along with other mods they
would need to have otherwise they wouldn't be able to join due to registrey
missing

**Comment by itsdinkd** [comment]
Registry*

**Comment by jacksonhardaway** [comment]
The Pollen login packet comes before the registry sync afaik, so they would be kicked because of this issue first. The only person causing this error is "jurassicking". Double check they ACTUALLY have Pollen loaded

**Comment by itsdinkd** [comment]
I am not talking about pollen having a registry. I am talking about the other mods that I have added into the update I released with pollen. If pollen did not exist in the modpack when they joined the server, that would
mean they do not have the other mods that do have a registry, and would be unable to join.


Not sure why I have to argue about this, there are other people that have
joined since that produce the same error.
